subjects/unzipstring/README.md

```diff
@@ -0,0 +1,61 @@
+## Unzipstring
+
+
+### Instructions
+
+Unzipstring is a simple function that takes a string in of form of a number and an alphabet like (2a) and returns the original string (aa).
```

Comment:
you can change this line with this : 
> write a function named `Unz...` that takes a string ...
and  I guess there is no need to add an example in the instructions part.